lib/profilefunctions.js

```diff
@@ -36,30 +37,41 @@ function init(profileData) {
 	
 	profile.getValueByTime = function getValueByTime (time, valueContainer) {
       if (!time) { time = new Date(); }
+      var tz = 'UTC';
+      if (profile.getTimezone()) tz = profile.getTimezone();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'tz'.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/768506899/issues/source?bid=2452379&fileBranchId=2457543#l41)